src/Jackett.Common/Indexers/HDSpace.cs

```diff
@@ -102,90 +96,93 @@ public override async Task<IndexerConfigurationStatus> ApplyConfiguration(JToken
                         continue;
 
-                    var qRow = row.Cq();
                     var release = new ReleaseInfo();
-
                     release.MinimumRatio = 1;
                     release.MinimumSeedTime = 172800; // 48 hours
 
-                    var qLink = row.ChildElements.ElementAt(1).FirstElementChild.Cq();
+                    //var qLink = row.ChildElements.ElementAt(1).FirstElementChild.Cq();
+                    var qLink = row.Children[1].FirstElementChild;
                     release.Title = qLink.Text().Trim();
-                    release.Comments = new Uri(SiteLink + qLink.Attr("href"));
+                    release.Comments = new Uri(SiteLink + qLink.GetAttribute("href"));
                     release.Guid = release.Comments;
 
-                    var qDownload = row.ChildElements.ElementAt(3).FirstElementChild.Cq();
-                    release.Link = new Uri(SiteLink + qDownload.Attr("href"));
+                    //var qDownload = row.ChildElements.ElementAt(3).FirstElementChild.Cq();
+                    var qDownload = row.Children[3].FirstElementChild;
+                    release.Link = new Uri(SiteLink + qDownload.GetAttribute("href"));
 
                     //"July 11, 2015, 13:34:09", "Today at 20:04:23"
-                    var dateStr = row.ChildElements.ElementAt(4).Cq().Text().Trim();
-                    if (dateStr.StartsWith("Today"))
-                        release.PublishDate = DateTime.Today + TimeSpan.ParseExact(dateStr.Replace("Today at ", ""), "hh\\:mm\\:ss", CultureInfo.InvariantCulture);
-                    else if (dateStr.StartsWith("Yesterday"))
-                        release.PublishDate = DateTime.Today - TimeSpan.FromDays(1) + TimeSpan.ParseExact(dateStr.Replace("Yesterday at ", ""), "hh\\:mm\\:ss", CultureInfo.InvariantCulture);
-                    else
-                        release.PublishDate = DateTime.SpecifyKind(DateTime.ParseExact(dateStr, "MMMM dd, yyyy, HH:mm:ss", CultureInfo.InvariantCulture), DateTimeKind.Local);
-
-                    var sizeStr = row.ChildElements.ElementAt(5).Cq().Text();
+                    //var dateStr = row.ChildElements.ElementAt(4).Cq().Text().Trim();
+                    var dateStr = row.Children[4].Text().Trim();
+                    //if (dateStr.StartsWith("Today"))
+                    //    release.PublishDate = DateTime.Today + TimeSpan.ParseExact(dateStr.Replace("Today at ", ""), "hh\\:mm\\:ss", CultureInfo.InvariantCulture);
+                    //else if (dateStr.StartsWith("Yesterday"))
+                    //    release.PublishDate = DateTime.Today - TimeSpan.FromDays(1) + TimeSpan.ParseExact(dateStr.Replace("Yesterday at ", ""), "hh\\:mm\\:ss", CultureInfo.InvariantCulture);
+                    //else
+                    //    release.PublishDate = DateTime.SpecifyKind(DateTime.ParseExact(dateStr, "MMMM dd, yyyy, HH:mm:ss", CultureInfo.InvariantCulture), DateTimeKind.Local);
```

Comment:
I assume that `DateTimeUtil.FromUnknown` handles all cases. Is that right?

Reply:
Yes, FromUnknown (https://github.com/Jackett/Jackett/blob/28b53d8fb1b6161c69ae5b76e68d6f3b66aedcbd/src/Jackett.Common/Utils/DateTimeUtil.cs#L130) first checks for "today"/"tomorrow", removes it and checks for a valid time. It doesn't check/remove the "at" though, which is why my replacement removes it before passing. If it finds no other special times, it just does a normal parse.

All three cases in this tracker work as expected when I tested with the new code.